controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -145,7 +145,23 @@ def run_match(supervisor: Supervisor) -> None:
         robot.getField('customData').setSFString('start')
 
     # ... then un-pause the simulation, so they all start together
-    supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_RUN)
+    if supervisor.getDevice("2021a-compatibility") is None:
+        # we are running version 2020b so the old command is used
+        supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_RUN)
+    else:
+        # webots-2021a removed the RUN mode and now uses FAST
+        print(
+            "All official competition matches will be run on "
+            "Webots version 2020b-rev1 unless otherwise stated",
+            file=sys.stderr,
+        )
+        print(
+            "Since this simulator is running a different version "
+            "it is possible that some behaviour may not match "
+            "that of the official competition matches",
+            file=sys.stderr,
+        )
+        supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_FAST)
```

Comment:
Would be great to pull this out to it's own util. The rest of this function is fairly concentrated on just the things needed to actually run the match, yet this block has quite a lot of detail which isn't really relevant to that.

This block ending up something like this might be good?
```python
    supervisor.simulationSetMode(get_simulation_run_mode())
```
Alternatively just having this block within a function like this would be good:
```python
    set_simulation_run_mode(supervisor)
```

Reply:
> Would be great to pull this out to it's own util. The rest of this function is fairly concentrated on just the things needed to actually run the match, yet this block has quite a lot of detail which isn't really relevant to that.

We could pull this out into a function but it should remain within the competition_supervisor since it operates directly upon the supervisor's node.

> This block ending up something like this might be good?
> 
> ```python
>     supervisor.simulationSetMode(get_simulation_run_mode())
> ```

I prefer this format since the overall parameter can be referred to as the run mode so a setter is quite ambiguous.

Reply:
One small issue, what should the return type of the getter be since in our stubs we've created the _SimulationMode enum while in webots it appear to give the parameter directly to SWIG which is an enum in C.

Reply:
Hrm. I wouldn't want to make `_SimulationMode` public as that would imply that it could be used directly (which it obviously can't be).

Reply:
Also when running in webots the type would be `int` and `_SimulationMode` would be undefined

Reply:
I think the solution here is probably to use a `NewType`. It's not ideal, as we'll need to import the type definition with a type check guard, but it preserves the rest of the safety we currently get from the enum.

Reply:
I've put together a fix in https://github.com/srobo/competition-simulator/pull/262

Reply:
5ba0d8d00ac13a395d7b6a2ae60c804220d7d309